tests/run_pretrained_models.yaml

```diff
@@ -454,3 +454,30 @@ ssd_mobilenet_v2_300_float_tflite:
     - TFLite_Detection_PostProcess:1
     - TFLite_Detection_PostProcess:2
     - TFLite_Detection_PostProcess:3
+
+deeplabv3_mnv2_ade20k_float_tflite:
+  tf_min_version: 2.1
+  disabled: false
+  url: https://github.com/mlcommons/mobile_models/raw/main/v0_7/tflite/deeplabv3_mnv2_ade20k_float.tflite
+  model: "deeplabv3_mnv2_ade20k_float.tflite"
+  model_type: tflite
+  input_get: get_ade20k
+  ptol: 0.001
+  inputs:
+    "MobilenetV2/MobilenetV2/input": [1, 512, 512, 3]
+  outputs:
+    - ArgMax
+
+deeplabv3_mnv2_ade20k_uint8_tflite:
+  tf_min_version: 2.1
+  disabled: false
+  url: https://github.com/mlcommons/mobile_models/raw/main/v0_7/tflite/deeplabv3_mnv2_ade20k_uint8.tflite
+  model: "deeplabv3_mnv2_ade20k_uint8.tflite"
+  model_type: tflite
+  input_get: get_ade20k_uint8
+  ptol: 1.0
```

Comment:
This is a 1% error rate which is surprisingly high.  I don't know why it is so high for the quantizized model, probably because ORT doesn't have quantization implemented for some ops so the ORT result may be slower but actually more accurate than (but different from) the tf results.